src/utils/icons.ts

```diff
@@ -91,6 +91,19 @@ export default {
 </script>`
 }
 
+export function SvgToVueTs(svg: string, name: string) {
+  return `
+<template>
+  ${ClearSvg(svg)}
+</template>
+
+<script lang='ts'>
+export default {
+  name: '${name}'
+}
+</script>`
+}
+
```

Comment:
For `js(x)/ts(x)`, their templates are not much different. I prefer to use the same function to control the generation of templates through variables. It also avoids memory overhead, doesn't it? What do you think?

Reply:
Do you mean to reuse the `SvgToVue` method and add a parameter like `isTs` to judge and do different processing?

Reply:
Ok, I'll update it.